packages/turndown-plugin-gfm/src/tables.js

```diff
@@ -98,10 +98,23 @@ rules.table = {
       }
     }
 
-    return '\n\n' + emptyHeader + content + '\n\n'
+    const captionContent = node.caption == null ? '' : node.caption.textContent || '';
```

Comment:
just `!note.caption`

Reply:
okay, will change
I usually try to avoid implicit boolean casting (and have ESLint rules for that) to write more explicit, less flaky code

Reply:
You previously had a non-strict equality check which is flaky anyway. But in general it's not useful to strictly check for null, undefined, empty string, etc. and that might even introduce bugs. There are cases where it matters but not here. Here we just want to check if something exists and it doesn't matter if it's null, undefined, false or something else.